From: Sorah Fukumori <her@sorah.jp>
Date: Sun, 5 Feb 2023 06:25:03 +0900
Subject: extract-gems-sequential: Keep using RUNRUBY

1. sorah-rbpkg's modified rubyN.M-gems requires to kick
   extract-gems and our build environment has no BASERUBY.

   Partially reverts https://github.com/ruby/ruby/pull/6203

2. For the same reason, partially reverting
   https://github.com/ruby/ruby/pull/7347
   to avoid calling git for extracting gems.
---
 common.mk | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/common.mk b/common.mk
index 0ea360c..9b34859 100644
--- a/common.mk
+++ b/common.mk
@@ -1537,7 +1537,7 @@ update-gems$(gnumake:yes=-sequential): PHONY
 
 extract-gems$(gnumake:yes=-sequential): PHONY
 	$(ECHO) Extracting bundled gem files...
-	$(Q) $(BASERUBY) -C "$(srcdir)" \
+	$(Q) $(RUNRUBY) -C "$(srcdir)" \
 	    -Itool/lib -rfileutils -rbundled_gem -answ \
 	    -e 'BEGIN {d = ".bundle/gems"}' \
 	    -e 'gem, ver, _, rev = *$$F' \
@@ -1551,8 +1551,6 @@ extract-gems$(gnumake:yes=-sequential): PHONY
 	    -e 'end' \
 	    gems/bundled_gems
 
-extract-gems$(gnumake:yes=-sequential): $(HAVE_GIT:yes=clone-bundled-gems-src)
-
 clone-bundled-gems-src: PHONY
 	$(Q) $(BASERUBY) -C "$(srcdir)" \
 	    -Itool/lib -rbundled_gem -answ \
